.github/workflows/unit-tests.yaml

```diff
@@ -0,0 +1,45 @@
+name: Unit Tests
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  workflow_dispatch:
```

Comment:
so this `workflow dispatch` allows us to manually run it?

Reply:
Yes, and we can also manually run it on any branch.